echo-web/config/echo.yml

```diff
@@ -25,6 +33,20 @@ webhooks:
     enabled: false
     sources: []
 
+scheduler:
+  enabled: ${services.echo.cron.enabled:true}
+  threadPoolSize: 20
```

Comment:
This defaults to 16 in code; I'd rather just remove it from the config file and fall back on that default.